Adds bounty page to docs site (#493)
onflow/flow: docs/content/bounties/responsible-disclosure.md

---
title: Responsible Disclosure Policy
description: Policy and guidelines for submitting security related bugs.
---

<div>

>   <button>Submit a Bug</button>

Author: See above coment.

- Testing of applications or systems NOT covered by the ‘In Scope’ section, or that are explicitly out of scope.
- Network level Denial of Service (DoS/DDoS) attacks

### Sensitive Data

In the interests of protecting privacy, we never want to receive:

- Personally identifiable information (PII)
- Payment card (e.g. credit card) data
- Financial information (e.g. bank records)
- Health or medical information
- Accessed or cracked credentials in cleartext

## Our Commitment To You
  
If you follow these guidelines when researching and reporting an issue to us, we commit to:
  
- Not send lawyers after you related to your research under this policy;
- Work with you to understand and resolve any issues within a reasonable timeframe, including an initial confirmation of your report within 72 hours of submission; and
- At a minimum, we will recognize your contribution in our Disclosure Acknowledgements if you are the first to report the issue and we make a code or configuration change based on the issue.

## Disclosure Acknowledgements
  
  Security acknowledgements can be found at https://dapperlabs.com/security_nods.txt

## Reporting Security Findings

> Reports welcome! Please do reach out to us if you have a security concern. If you believe you may have found a security vulnerability in one of our products or platforms, send us an email: [security@dapperlabs.com](mailto:security@dapperlabs.com?subject=Responsible%20disclosure)
> 

Author: ```suggestion
Reports welcome! Please do reach out to us if you have a security concern. If you believe you may have found a security vulnerability in one of our products or platforms, send us an email: [security@dapperlabs.com](mailto:security@onflow.org?subject=Responsible%20disclosure)

```

> We encourage you to encrypt the information you send us using our PGP key at https://dapperlabs.com/security@dapperlabs.com.pgp.txt

Reviewer: Not sure where this is from, but the email and key should be the ones as found here: 
https://github.com/onflow/cadence/blob/master/SECURITY.md#reportingsecurityfindingstous

> We encourage you to encrypt the information you send us using our PGP key at https://dapperlabs.com/security@dapperlabs.com.pgp.txt

Author: ```suggestion
We encourage you to encrypt the information you send us using our PGP key at [keys.openpgp.org/security@onflow.org](keys.openpgp.org/security@onflow.org)

```

---
title: Responsible Disclosure Policy
description: Policy and guidelines for submitting security related bugs.
---

> <div>
>   <button>Submit a Bug</button>
> </div>

Author: ```suggestion
If you believe you may have found a security vulnerability in one of our products or platforms, send us an email: [security@dapperlabs.com](mailto:security@onflow.org?subject=Responsible%20disclosure)
```

- Accessed or cracked credentials in cleartext

## Our Commitment To You
  
If you follow these guidelines when researching and reporting an issue to us, we commit to:
  
- Not send lawyers after you related to your research under this policy;
- Work with you to understand and resolve any issues within a reasonable timeframe, including an initial confirmation of your report within 72 hours of submission; and
- At a minimum, we will recognize your contribution in our Disclosure Acknowledgements if you are the first to report the issue and we make a code or configuration change based on the issue.

## Disclosure Acknowledgements
  
  Security acknowledgements can be found at https://dapperlabs.com/security_nods.txt

## Reporting Security Findings

Reports welcome! Please do reach out to us if you have a security concern. If you believe you may have found a security vulnerability in one of our products or platforms, send us an email: [security@dapperlabs.com](mailto:security@dapperlabs.com?subject=Responsible%20disclosure)

We encourage you to encrypt the information you send us using our PGP key at [keys.openpgp.org/security@onflow.org](keys.openpgp.org/security@onflow.org)

> [🔐 PGP Public Key](https://www.notion.so/PGP-Public-Key-dc945aa551904be0a8d0a48b44e6bf68)

Author: ```suggestion

```
